.travis.yml

```diff
@@ -113,6 +113,31 @@ matrix:
        - mkdir -p $HOME/.cargo/bin
        - cp cargo-dinghy-macos/cargo-dinghy $HOME/.cargo/bin
 
+    - name: cargo build --release (stable, android)
+      os: linux
+      rust: stable
+      env: CI_STAGE_BUILD_RELEASE=yes CI_ANDROID_TEST=yes JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64 ANDROID_SDK_ROOT=/opt/android-ndk-r21d
+      before_script:
+        # aarch64 and armv7 cover most Android phones & tablets.
+        # However armv7 is not yet available for Rust stable.
```

Comment:
^ actually that comment is wrong, I just realized it's me doing a dumb typo when trying to compile for armv7. It should be `armv7-linux-androideabi` instead of `armv7-linux-android` (I forgot the `eabi`). What's the best option here, removing the comment or adding `armv7 -linux-androieabi` on top of the existing option ?

Reply:
Maybe both if you can reuse the SDK download?

Reply:
OK, I added this commit https://github.com/godot-rust/godot-rust/pull/649/commits/6fa40032155cd7bf8bf71e81e850ee0b39874663 which should indeed build it for
 armv7. Waiting now for Travis to tell me whether it breaks anything.